pkg/api/v1/control-plane/server/generic_server.go

```diff
@@ -21,54 +21,54 @@ import (
 	"strconv"
 	"sync/atomic"
 
-	envoy_api_v2 "github.com/envoyproxy/go-control-plane/envoy/api/v2"
 	envoy_config_core_v3 "github.com/envoyproxy/go-control-plane/envoy/config/core/v3"
-	envoy_service_discovery_v2 "github.com/envoyproxy/go-control-plane/envoy/service/discovery/v2"
 	envoy_service_discovery_v3 "github.com/envoyproxy/go-control-plane/envoy/service/discovery/v3"
 	"github.com/golang/protobuf/proto"
 	"github.com/golang/protobuf/ptypes/any"
+	sk_discovery "github.com/solo-io/solo-kit/pkg/api/external/envoy/api/v2"
 	"github.com/solo-io/solo-kit/pkg/api/v1/control-plane/resource"
 	"github.com/solo-io/solo-kit/pkg/api/v1/control-plane/util"
+	solo_discovery "github.com/solo-io/solo-kit/pkg/api/xds"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 
 	"github.com/solo-io/solo-kit/pkg/api/v1/control-plane/cache"
 )
 
-type StreamV3 interface {
+type StreamEnvoyV3 interface {
```

Comment:
I decided to also change `StreamV3` to `StreamEnvoyV3` since it provides more context onto what is V3